react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,43 @@
+.hidden {
+  display: none;
+}
+
+.form {
+  margin: 40px;
+}
+
+.password-label {
+  font-family: 'Patrick Hand', sans-serif;
+  font-size: 24px;
+  letter-spacing: 1px;
+}
+
+.password-input {
+  width: 200px;
+  height: 30px;
+  font-size: 34px;
+  padding-left: 7px;
+  margin-top: 7px;
+  border: 3px solid black;
+  outline-color: black;
+}
+
+.fa-times,
+.fa-check {
+  font-size: 32px;
+  margin-left: 18px;
+}
+
+.fa-times {
+  color: red;
+}
+
+.fa-check {
+  color: green;
+}
```

Comment:
You shouldn't add CSS rules to existing CSS classes from CSS libraries. This can cause issues in other parts of your app and/or cause confusion with other people working on the project because the classes aren't behaving as expected and outlined in the documentation. So a better solution is to make your own classes and use those where needed. For example:

```CSS
.icon {
  font-size: 32px;
  margin-left: 18px;
}

.icon-red {
  color: red;
}

.icon-green {
  color: green;
}
```